benefits/eligibility/views.py

```diff
@@ -100,7 +100,10 @@ def start(request):
         details=_(verifier.start_item_details),
     )
 
+    help_anchor = "mst-courtesy-card"
+
     if verifier.is_auth_required:
+        help_anchor = "login-gov"
```

Comment:
@thekaveman I wasn't sure what a better way to do this would be.

Reply:
I think this should probably be added as a property to the `Verifier` model and not hardcoded. Will help us when adding another Agency later.